WSL/compare-versions.md

```diff
@@ -92,4 +86,16 @@ We recommend that you use WSL 2 as it offers faster performance and 100% system
 - If you rely on a Linux distribution to have an IP address in the same network as your host machine, you may need to set up a workaround in order to run WSL 2. WSL 2 is running as a hyper-v virtual machine. This is a change from the bridged network adapter used in WSL 1, meaning that WSL 2 uses a Network Address Translation (NAT) service for it's virtual network, instead of making it bridged to the host Network Interface Card (NIC) resulting in a unique IP address that will change on restart. To learn more about the issue and workaround that forwards TCP ports of WSL 2 services to the host OS, see [WSL GitHub repository issue 4150, NIC Bridge mode (TCP Workaround)](https://github.com/microsoft/WSL/issues/4150).
 
 > [!NOTE]
-> Consider trying the VS Code [WSL Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl) to enable you to store your project files on the Linux file system, using Linux command line tools, but also using VS Code on Windows to author, edit, debug, or run your project in an internet browser without any of the performance slow-downs associated with working across the Linux and Windows file systems. [Learn more](tutorials/wsl-vscode.md).
+> Consider trying the VS Code [Remote WSL Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl) to enable you to store your project files on the Linux file system, using Linux command line tools, but also using VS Code on Windows to author, edit, debug, or run your project in an internet browser without any of the performance slow-downs associated with working across the Linux and Windows file systems. [Learn more](tutorials/wsl-vscode.md).
+
+## WSL in the Microsoft Store and WSL as a Windows component
```

Comment:
Proposing a rewrite to this in hopes of reducing confusion... 

## WSL Preview in the Microsoft Store



WSL is included in the Windows operating system as optional components that need to be enabled in order to install a Linux distribution. That process of enabling WSL components and installing a Linux distribution has been simplified into a single [wsl.exe --install command](./install.md). Beginning in Windows version 22000 or higher, running the wsl --install command will install the WSL Preview from the Microsoft Store. The WSL Preview version in the Store is the same WSL, with the same user experience, but receives updates and servicing as soon as they're available, rather than needing to wait for an update of your Windows operating system. ([See the blog post announcing this update](https://devblogs.microsoft.com/commandline/a-preview-of-wsl-in-the-microsoft-store-is-now-available/)). If you are already using WSL, you can update to ensure that you're receiving the latest WSL features and servicing from the Store by running `wsl.exe --update`.